Makefile

```diff
@@ -84,6 +84,15 @@ remove: install/prepare
 test: build/test-suite
 	./build/test-suite
 
+.PHONY: fuzz
+fuzz: build/fuzz-classifier-spdx
+	afl-fuzz -i test/fuzz/input -o test/fuzz/output "$(PWD)/build/fuzz-classifier-spdx"
+
+.PHONY: fuzz-coverage
+fuzz-coverage: CFLAGS += --coverage
+fuzz-coverage: LDLIBS += -lgcov
```

Comment:
In other parts of the Makefile, we use `LDFLAGS` for holding `-llibrary` flags. Please either use `LDFLAGS` here or rename the current variable to `LDLIBS`.

Reply:
I ran into issues when `-lfoo` flags were defined before `.o` - in some cases, the linker may ignore libraries passed at command line, if no object files preceding it make use of them (see e.g. https://wiki.gentoo.org/wiki/Project:Quality_Assurance/As-needed#Importance_of_linking_order)